moveit_ros/planning/planning_scene_monitor/src/current_state_monitor.cpp

```diff
@@ -134,13 +136,79 @@ void planning_scene_monitor::CurrentStateMonitor::clearUpdateCallbacks()
   update_callbacks_.clear();
 }
 
+bool planning_scene_monitor::CurrentStateMonitor::getIsRobotStopped(const std::string& group, double joint_velocity_tol,
+                                                                    double state_wait_time, bool& is_stopped)
+{
+  // Initially assume the robot is not stopped
+  is_stopped = false;
+
+  // Constant joint group variables
+  const auto& joint_models = robot_model_->getActiveJointModels();
+  const std::size_t num_joints = joint_models.size();
```

Comment:
This should be the number of joints of the JointModelGroup, not of the full robot model.